src/main.rs

```diff
@@ -127,6 +129,23 @@ fn main() {
             std::process::exit(exitcode::CONFIG);
         });
 
+        // API
+        #[cfg(feature = "api")]
+        if config.api.enabled {
+            let opt = config.api; // copy to move to spawn
+            tokio::spawn(async move {
+                let addr = opt.bind.unwrap();
+                let playground = &*format!("http://{}:{}/playground", addr.ip(), addr.port());
+
+                info!(
```

Comment:
Below, on line 155, we `emit!(VectorStarted)`, maybe here would be a good place to mimic this?

Reply:
Added an `ApiStarted` event in 80bd6615e0290a75855a22bae2e1d404f0e2f8eb